python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_decorator.py

```diff
@@ -463,12 +463,12 @@ def my_dbt_assets(): ...
     assert len(my_dbt_assets.keys) == len(expected_specs)
     assert my_dbt_assets.keys == {spec.key for spec in expected_specs}
     assert my_dbt_assets.keys_by_input_name == {
-        "__subset_input__model_jaffle_shop_stg_customers": AssetKey(["prefix", "stg_customers"]),
-        "__subset_input__model_jaffle_shop_stg_orders": AssetKey(["prefix", "stg_orders"]),
-        "__subset_input__model_jaffle_shop_stg_payments": AssetKey(["prefix", "stg_payments"]),
-        "__subset_input__seed_jaffle_shop_raw_customers": AssetKey(["prefix", "raw_customers"]),
-        "__subset_input__seed_jaffle_shop_raw_orders": AssetKey(["prefix", "raw_orders"]),
-        "__subset_input__seed_jaffle_shop_raw_payments": AssetKey(["prefix", "raw_payments"]),
+        "__subset_input__prefix__stg_customers": AssetKey(["prefix", "stg_customers"]),
```

Comment:
why is this changing?

Reply:
see description -- basically when we were constructing the `outs` dictionary directly, we had a scheme for converting from asset key to valid python identifier that was distinct from `AssetKey.to_python_identifier()` (`dagster_name_fn`).

When we just pass in a list of specs, we no longer have control over that process (the framework automatically calls `to_python_identifier`), so these tests that depended on the specific identifier needed to be updated

Reply:
Okay that makes sense - I'm trying to think through whether the behavior change here might cause any issues.

I think it's fine, since this isn't used for any type of historical data retrieval right. Like this value is serialized, but it's only used in passing snapshots over the wire.